web_frontend/osmaxx/excerptexport/forms/order_options_mixin.py

```diff
@@ -6,11 +6,11 @@
 
 # TODO: fetch from API
 available_format_choices = (
-    ('fgdb', _('fgdb')),
-    ('shp', _('shp')),
-    ('gpkg', _('gpkg')),
-    ('spatialite', _('spatialite')),
-    ('garmin', _('garmin')),
+    ('fgdb', _('ESRI File Geodatabase (FileGDB)')),
+    ('shp', _('ESRI Shapefile (shp)')),
+    ('gpkg', _('GeoPackage (gpkg)')),
+    ('spatialite', _('SQLite based SpatiaLite (spatialite)')),
+    ('garmin', _('Garmin navigation & map data (img, tdb)')),
```

Comment:
All strings that are not marked with `mark_safe` [and similar](https://docs.djangoproject.com/en/1.9/ref/utils/#module-django.utils.html) are being escaped.